starknet/src/L1_headers_store/contract.cairo

```diff
@@ -12,12 +12,14 @@ pub mod L1HeaderStore {
     use fossil::library::blockheader_rlp_extractor::{
         decode_parent_hash, decode_uncle_hash, decode_beneficiary, decode_state_root,
         decode_transactions_root, decode_receipts_root, decode_difficulty, decode_base_fee,
-        decode_timestamp, decode_gas_used
+        decode_timestamp, decode_gas_used, decode_block_number
     };
+    use fossil::library::keccak256::keccak256;
```

Comment:
Here we can group the imports to use less lines, eg: all the imports from `library` with only one use, same for the others